<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixGeometryWidgetForm</class>
 <widget class="QWidget" name="FixGeometryWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="m_inputGroupBox">
       <property name="title">
        <string>Input layer</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QComboBox" name="m_layersComboBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="m_logGroupBox">
       <property name="title">
        <string>Log</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLineEdit" name="m_repositoryLineEdit_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="m_logToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Repository:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="m_repositoryLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="m_targetFileToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="m_targetDatasourceToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>BD</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Layer Name:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="m_newLayerNameLineEdit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
